recipes/botan/all/conanfile.py

```diff
@@ -0,0 +1,320 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanException, ConanInvalidConfiguration
+from conans.model.version import Version
+
+
+class BotanConan(ConanFile):
+    name = 'botan'
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/randombit/botan"
+    license = "BSD 2-clause"
+    exports = ["patches/*"]
+    description = "Botan is a cryptography library written in C++11."
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    options = {
+        'amalgamation': [True, False],
+        'bzip2': [True, False],
+        'debug_info': [True, False],
+        'openssl': [True, False],
+        'quiet': [True, False],
+        'shared': [True, False],
+        'fPIC': [True, False],
+        'single_amalgamation': [True, False],
+        'sqlite3': [True, False],
+        'zlib': [True, False],
+        'boost': [True, False],
+        'enable_modules': "ANY",
+        'system_cert_bundle': "ANY"
+    }
+    default_options = {'amalgamation': True,
+                       'bzip2': False,
+                       'debug_info': False,
+                       'openssl': False,
+                       'quiet': True,
+                       'shared': True,
+                       'fPIC': True,
+                       'single_amalgamation': False,
+                       'sqlite3': False,
+                       'zlib': False,
+                       'boost': False,
+                       'enable_modules': None,
+                       'system_cert_bundle': None}
+
+    def configure(self):
+        msvc_too_old = self.settings.os == "Windows" and \
+                       self.settings.compiler == "Visual Studio" and \
+                       Version(self.settings.compiler.version.value) < "14"
+        if msvc_too_old:
+            raise ConanInvalidConfiguration("Botan doesn't support MSVC < 14")
+
+        if self._needs_cxx_abi_workaround:
+            self.output.warn("Overriding compiler.libcxx setting to use libstdc++11")
+            self.settings.compiler.libcxx = 'libstdc++11'
+
+        if self.options.boost:
+            self.options["boost"].add("shared=False")
+            self.options["boost"].add("magic_autolink=False")
+            self.options["boost"].add("without_coroutine=False")
+            self.options["boost"].add("without_system=False")
+
+    def build_requirements(self):
+        if self.settings.os == "Windows":
+            self.build_requires("jom_installer/1.1.2@bincrafters/stable")
+
+    def requirements(self):
+        if self.options.bzip2:
+            self.requires("bzip2/1.0.6")
+        if self.options.openssl:
+            self.requires("OpenSSL/1.0.2s")
```

Comment:
Apparently, this is based on an older version of our recipe as our recipe doesn't contain this mistake 🤔 

```suggestion
            self.requires("openssl/1.0.2t")
```

Reply:
You're right, the `testing/2.12.1` branch uses lower-case already. I based it on the `stable/2.12.1` branch. Lucky for me it's almost the only difference between the testing and stable recipe.